configure

```diff
@@ -1121,25 +1182,34 @@ sub win32_cmake_generator {
   return $map{$ver} . ($arch eq 'x64' ? ' Win64' : '');
 }
 
-## Google Test
-if ($opts{'security'} && $tests && $opts{'cmake'}) {
+## Build Google Test
+if ($opts{'security'} && $tests && $opts{'cmake'} && ! $opts{'gtest-precompiled'}) {
   my $cwd = Cwd::getcwd();
-  my $build_dir = 'tests' . $slash . 'googletest' . $slash . 'build';
+  my $build_dir = $opts{'gtest'} . $slash . 'build';
+  my $install_dir = $build_dir . $slash . 'install';
   my $cmake = $opts{'cmake'};
   my @cmake_cmds = ((is_windows() ?
                       ["$cmake",
                        "-D", "gtest_force_shared_crt=ON",
                        "-D", "CMAKE_CXX_FLAGS=/D_SILENCE_TR1_NAMESPACE_DEPRECATION_WARNING=1",
+                       "-D", "CMAKE_INSTALL_PREFIX=$install_dir",
```

Comment:
Perhaps this is redundant with the earlier discussion, but what do we gain by doing another step here (adding complexity)?

Reply:
Along the lines of our earlier discussion, instead of depending upon the Google Test source tree, sub-modules will depend upon the include/lib directory of the compiled sources (which is Google Test is used in the system packages).

Reply:
Within this conditional block we know we're not using a pre-built or system-installed Google Test, since we are actually building it.

Reply:
True, however, within the googletest.mpb having one standard include/lib directory based on the GTEST_ROOT is much cleaner, instead of having dependencies on the googletest sources. Suppose Google Test decided to move their files around in the sources (which they are free to do) then we would have to make a change; by using this install directory we guard against such changes in the future.

Reply:
The submodule setup guarantees we get version 1.8.  If a user doesn't use submodules, our documentation states we're compatible with 1.8.

Reply:
The benefit of being correct/future-proof outweighs the small (and temporary) complexity added. Especially given that the code change is already in! :)